partials/tabs/mentors.html

```diff
@@ -47,7 +47,7 @@
             <div  ng-repeat="member in gic.adminsList" class="card card-main showcase-card-gi">
                 <div class="empty"></div>
                 <div class="card-action profile-data" >
-                  <center><img class="profile-img" height="100" ng-src="{{ member.github_avatar_url }}"></center>
+                  <img class="profile-img" height="100" ng-src="{{ member.github_avatar_url }}">
```

Comment:
why not just apply the center-block class? Should be avail by default in bootstrap